src/java.base/share/classes/sun/security/ec/ECPublicKeyImpl.java

```diff
@@ -23,12 +23,6 @@
  * questions.
  */
 
-/*
```

Comment:
This file should be reverted so it is identical to the content in 31fbd9636b333a9e23366d59fbbe2d84186f6b16.